section/doc/tactic-developer/developer/watch-folder-service/index.txt

```diff
@@ -7,26 +7,33 @@ Currently the implementation will create an entry per file.  Subsequent drops of
-Under the Admin Views section, go to Schema Views > Global Config/Data and open the "Watch Folder" view.
-Add an entry by selecting the "+" (plus) button in the shelf in the view.
-In this view there will be 4 columns you will need to fill in to define the watch folder:
-Project Code - The name of the project you are working in i.e. sample_project
-Base Dir - The name of the directory that will be "watched". This is where you drop the files to be checked in to TACTIC. i.e. /home/tactic/drop. 
-Search Type - The sType or table that the assets or files will be checked in to i.e. sample_project/media
-Process (optional) - The pipeline process that the asset will be checked into. By default, the asset is checked into the "publish" process, if no process is specified. This is sufficient when no pipeline processes have been defined, or the asset doesn't need to be checked into a certain process. Otherwise, the process needs to be specified i.e. publish
-
-Save the entry by selecting the floppy disk icon in the shelf.
-If a new Watch Folder entry is added, restart the TACTIC service. To do this, if you SSH into the server and you are logged in as "root", navigate to the TACTIC install and type, "service tactic restart". This will restart the TACTIC service and the Watch Folder will be ready for use.
+1.  Go to the Administrative layer of your TACTIC project by selecting the Admin bar at the top of the page after signing in. Only users in the Administrative group or users with Admin permissions will have the Admin bar and have access to the Administrative layer.
+2.  Open the sidebar.
+3.  Under the Admin Views section, go to Schema Views > Global Config/Data and open the "Watch Folder" view.
+4.  Add an entry by selecting the "+" (plus) button in the shelf in the view.
+5.  In this view there will be 4 columns you will need to fill in to define the watch folder:
++
+a)  Project Code - The name of the project you are working in i.e. sample_project
++
+b)  Base Dir - The name of the directory that will be "watched". This is where you drop the files to be checked in to TACTIC. i.e. /home/tactic/drop. 
++
+c)  Search Type - The sType or table that the assets or files will be checked in to i.e. sample_project/media
++
+d)  Process (optional) - The pipeline process that the asset will be checked into. By default, the asset is checked into the "publish" process, if no process is specified. This is sufficient when no pipeline processes have been defined, or the asset doesn't need to be checked into a certain process. Otherwise, the process needs to be specified i.e. publish
+
+6.  Save the entry by selecting the floppy disk icon in the shelf.
+7.  If a new Watch Folder entry is added, restart the TACTIC service. To do this, if you SSH into the server and you are logged in as "root", navigate to the TACTIC install and type, "service tactic restart". This will restart the TACTIC service and the Watch Folder will be ready for use.
 
 Limitations
```

Comment:
Maybe underline or italicize this based on common subsection conventions throughout the rest of the docs.